pallets/gear-debug/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
 #![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::manual_inspect)]
```

Comment:
add an issue to remove this?

Reply:
I'll create, maybe we need to update substrate

Reply:
#4097